cfme/infrastructure/virtual_machines.py

```diff
@@ -667,6 +680,13 @@ def get_changes_to_fill(self, other_configuration):
                 changes['disks'].append({'action': 'delete', 'disk': disk, 'delete_backing': None})
             elif disk not in self.disks and disk in other_configuration.disks:
                 changes['disks'].append({'action': 'add', 'disk': disk})
+            elif disk in self.disks and disk in other_configuration.disks:
```

Comment:
[Optional]
I think that there could be just 'else:', as it's iterating through these two lists. It is already checked if a disk is just in one of the lists, so this condition should be always True (if the lists are not modified during this for loop)